lib/bamboo/adapters/mandrill_adapter.ex

```diff
@@ -11,24 +11,22 @@ defmodule Bamboo.MandrillAdapter do
 
       Here is the response:
 
-      #{inspect response}
+      #{inspect response, limit: :infinity}
 
 
-      Here are the params we sent:
+      Here are the params that were sent:
 
-      #{inspect Poison.decode!(params)}
+      #{inspect Poison.decode!(params), limit: :infinity}
       """
       %ApiError{message: message}
     end
   end
 
   def deliver(email, config) do
-    api_key = get_key(config)
-    params = email |> convert_to_mandrill_params(api_key) |> Poison.encode!
-    case request!(@send_message_path, params) do
-      %{status_code: status} = response when status > 299 ->
-        raise(ApiError, %{params: params, response: response})
-      response -> response
+    if email.to == [] and email.cc == [] and email.bcc == [] do
+      {:no_recipients, email}
+    else
+      send_email(email, config)
```

Comment:
Being able to pattern match on the result seems useful. As you said you would need to add `deliver!` and `deliver_async!` but that's probably fine.